src/vorta/utils.py

```diff
@@ -240,7 +241,43 @@ def sort_sizes(size_list):
     return final_list
 
 
-def pretty_bytes(size, metric=True, sign=False, precision=1):
+T = TypeVar("T")
+
+
+def clamp(n: T, min_: T, max_: T) -> T:
+    """Restrict the number n inside a range"""
+    return min(max_, max(n, min_))
```

Comment:
```suggestion
    return min(max_, max(n, min_))  # type: ignore
```

Reply:
ok I activated mypy in my emacs config, didn't know about that checker! I added the `int, float` specification to the typevar and now it doesn't complain